pkg/etcdcli/etcdcli.go

```diff
@@ -294,7 +295,16 @@ func (g *etcdClientGetter) UnhealthyMembers() ([]*etcdserverpb.Member, error) {
 	for _, member := range membersResp.Members {
 		if len(member.ClientURLs) == 0 {
 			unhealthyMembers = append(unhealthyMembers, member)
-			unstartedMemberNames = append(unstartedMemberNames, member.Name)
+			unstartedName := member.Name
+			if unstartedName == "" {
```

Comment:
should we update the condition on L296 to this? I believe missing member name is the indication of unstarted member as used by etcdcli[1].

1. https://sourcegraph.com/github.com/etcd-io/etcd@6aea6edc341a1e15cfb86b765c9b9c0423516cc7/-/blob/etcdctl/ctlv3/command/printer.go?subtree=true#L171